src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
I think the question depends on how we scope the feature.  If it's only faded text with no error list entry, then I don't think we need an option, but if it's something we make diagnostics for, I'm more leery, as people might be upset having a bunch of these appear in their error list when they upgrade VS.